distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -190,7 +190,7 @@ type Column
        Returns a column with results of comparing this column's elements against
        `other`.
 
-       ! Warnings
+       ! Error Conditions
 
          - If this operation results in comparing floating-point values for
            equality which is not recommended, a `Floating_Point_Equality`
```

Comment:
I know we want an unified form, but I'm wondering if 'Error Conditions' title is generic enough.

In this section (and many similar ones) we describe _only warnings_. They will never be promoted to errors because the operator does not have the `on_problems` setting. So it feels like 'Error Conditions' does not fully suit it. What if we could have some more generic name for this? `! Problem Handling` maybe? Or `! Errors and Warnings`?

Reply:
I went with Error Conditions as our most used. It will be rendered as Errors in the markdown.

Reply:
I think that's my point - if we show it as "Errors" but we are only mentioning 'light' warnings, it feels a bit mismatched. Do you think "Errors and Warnings" could work maybe?